src/main/java/org/carlmontrobotics/RobotContainer.java

```diff
@@ -48,8 +56,18 @@ private void setDefaultCommands() {
     //   () -> driverController.getRawButton(OI.Driver.slowDriveButton)
     // ));
   }
-  private void setBindingsDriver() {}
-  private void setBindingsManipulator() {}
+  private void setBindingsDriver() {
+    // 4 cardinal directions on arrowpad
+		// slowmode toggle on trigger
+		// 3 cardinal directions on letterpad
+  }
+  private void setBindingsManipulator() {
+		// intake/outtake on triggers
+    //Left trigger will be intake whilst right trigger will be outtake
+    //3 setpositions of arm on letterpad
+    //Up is Speaker, down is ground, right is Amp
+    //right joystick used for manual arm control
```

Comment:
inconsistent with matthew's code, please discuss with whoever is working on intakeshooter

Reply:
+1
@Rand0mAsianKid @sofiebudman